debian/package.json

```diff
@@ -3,12 +3,14 @@
   "version": "1.6.0",
   "description": "Discover and collect music on YouTube",
   "main": "index.js",
+  "homepage": "http://headsetapp.co",
   "scripts": {
     "start": "NODE_ENV=development NODE_PATH=. nf start",
     "electron": "./node_modules/.bin/electron .",
     "player": "./bin/player-window",
     "production": "NODE_ENV=production electron .",
-    "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build"
+    "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build",
+    "rebuild": "electron-rebuild"
```

Comment:
I've never heard of yarn until now. Like I mentioned, I'm a newbie at electron and javascript. I'll install it on my system and see how it goes/works